whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/AttributeParser.java

```diff
@@ -199,6 +199,10 @@ public InternetAddress parse(final String s) {
                 final String address = parsed[0].getAddress();
                 final String localPart = address.substring(0, address.indexOf('@'));
 
+                if (address.length() > 80){
```

Comment:
Define 80 in a static field like `MAXIMUM_LENGTH` to the EmailParser class.

Reply:
Done